packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -453,7 +453,7 @@ export class EpochTracker implements IPersistedFileCache {
 			return new NonRetryableError(
 				"Epoch mismatch",
 				DriverErrorType.fileOverwrittenInStorage,
-				{ driverVersion },
+				{ driverVersion, serverEpoch: epochFromResponse },
```

Comment:
Should we also add clientEpoch: this.fluidEpoch here (and remove it from checkForEpochError)? The code is hard to follow though